hello-core-logic/removeBoardMimicTrello.js

```diff
@@ -0,0 +1,33 @@
+//Write a function named 'removeBoard' that takes one argument, boardName, then, if a board with the name exists, remove
+//it and return, Board 'Hello' was removed. If no board with the passed name exists, it should return Board doesn't exist
+
+const hello = {
+    'Tester Board': {
+      'To Do': ['Laundry', 'Buy Apples', 'Pay Phone Bill'],
+      'Doing': ['Laundry', 'Studying Javascript', 'Studying HTML', 'Studying Ruby'],
+      'Done': ['Laundry']
+    },
+    'Dreams': {
+      'Wish List': ['Conquer the Seven Kingdoms', 'Get my baby back', 'My hand needs to chill']
+    },
+    'Cuisine': {
+      'Region':['African', 'European', 'Brazilian', 'Asian'] }
+};
+
+board = "Cuisine";
+
+function removeBoard(obj, boardName) {
+  let arr = Object.keys(obj);
+  arr.forEach(function(key) {
```

Comment:
No need for the loop here. It does not work as you suspect, again.

better solution:
1. check if board exists
2. if it does, delete it
```javascript
if (hello[boardName]) {
   delete hello[boardName];
}
```
3. if it doesn't, return error string
```javascript
else {
   return `Board '${boardName}' cannot be deleted because it doesn't exist!`;
}
```